tests/testthat/test-argufy_package.R

```diff
@@ -0,0 +1,32 @@
+{
+  temp_lib <- file.path(tempdir(), "lib")
+
+  old <- .libPaths()
+  .libPaths(temp_lib)
+  on.exit(.libPaths(old))
+  tools:::.install_packages("TestS4")
```

Comment:
I don't think this will go through CRAN. Btw. why is it better than `install.packages`?